torch/library.py

```diff
@@ -418,8 +419,8 @@ def impl_abstract(qualname, func=None, *, lib=None, _stacklevel=1):
 
     """
     source = torch._library.utils.get_source(_stacklevel + 1)
-    frame = inspect.stack()[_stacklevel]
-    caller_module = inspect.getmodule(frame[0])
+    frame = sys._getframe(_stacklevel)
```

Comment:
sys._getframe isn't public, is there a risk that Python changes this?

Reply:
Nvm, I see it over at https://docs.python.org/3/library/sys.html#sys._getframe -- it's just python-implementation dependent.